pkg/mailer/template.go

```diff
@@ -93,7 +93,7 @@ It is the end of the grace period after the project expiration and thus data acc
 
 More information on project expiration and quota :
 
-  - ProjectExpirationProcedure, see https://intranet.donders.ru.nl/uploads/media/20190624-ProjectExpirationProcedure-Rev3.pdf
+  - ProjectExpirationProcedure, see https://intranet.donders.ru.nl/uploads/media/DCCN_SOP_Expired_Project_Handling_v2.5_Jun_2024.pdf
```

Comment:
Perhaps extract this URL to a constant variable that you can refer to in each of these usages?